docs/snippets_test.py

```diff
@@ -260,7 +305,9 @@ def naive_convert_snippet_code_to_python_2(snippet):
     return snippet.replace('...', 'Ellipsis')
 
 
-def assert_code_snippet_runs_and_prints_expected(snippet: str, state: Dict):
+def assert_code_snippet_runs_and_prints_expected(snippet: str,
+                                                 state: Dict,
+                                                 line_number: int = -1):
```

Comment:
slightly more pythonic to use None for the default there?

Reply:
Done.